src/sinks/s3_common/config.rs

```diff
@@ -311,8 +311,11 @@ impl RetryLogic for S3RetryLogic {
     type Error = SdkError<PutObjectError, HttpResponse>;
     type Response = S3Response;
 
-    fn is_retriable_error(&self, error: &Self::Error) -> bool {
-        is_retriable_error(error)
+    fn is_retriable_error(&self, _error: &Self::Error) -> bool {
```

Comment:
Is there any limit to how many retries we can do? Worried that we will have infinite retries which will make vector unstable.

Reply:
Merged a change to set limit